python_modules/libraries/dagster-postgres/dagster_postgres/event_log/event_log.py

```diff
@@ -202,11 +203,36 @@ def store_event(self, event: EventLogEntry) -> None:
                     "Cannot store asset event tags for null event id."
                 )
 
-            self.store_asset_event_tags(event, event_id)
+            self.store_asset_event_tags([event], [event_id])
 
         if event.is_dagster_event and event.dagster_event_type in ASSET_CHECK_EVENTS:
             self.store_asset_check_event(event, event_id)
 
+    def store_event_batch(self, events: Sequence[EventLogEntry]) -> None:
```

Comment:
I tried this in my initial implementation and had problems with sqlite. Will give it another shot.

Reply:
if thats the case may be best to override in sqllite specifically citing the reason it doesn't work there

Reply:
Tests confirm: neither MySQL nor SQLite support `RETURNING ...`, which is used in `store_event_batch`. There must be a way to get the same functionality but I think it would be best to address in a follow-up.

Reply:
got it, makes sense. Maybe slap a comment somewhere in case said follow-up does not manifest soon